content/docs/quickstart/csharp.md

```diff
@@ -54,26 +51,28 @@ From the `examples/csharp/Helloworld` directory:
 > dotnet build Greeter.sln
 ```
 
-*NOTE: If you want to use gRPC C# from a project that uses the "classic" .csproj files (supported by Visual Studio 2013, 2015 and older versions of Mono), please refer to the
-[Greeter using "classic" .csproj](https://github.com/grpc/grpc/blob/{{< param grpc_release_tag >}}/examples/csharp/HelloworldLegacyCsproj/README.md) example.*
+{{< note >}}
+If you want to use gRPC C# from a project that uses the "classic" .csproj files (supported by Visual Studio 2013, 2015 and older versions of Mono), please refer to the
+[Greeter using "classic" .csproj](https://github.com/grpc/grpc/blob/{{< param grpc_release_tag >}}/examples/csharp/HelloworldLegacyCsproj/README.md) example.
+{{< /note >}}
 
 ### Run a gRPC application
 
 From the `examples/csharp/Helloworld` directory:
 
 * Run the server:
 
-```sh
-> cd GreeterServer
-> dotnet run -f netcoreapp2.1
-```
+    ```sh
+    > cd GreeterServer
+    > dotnet run -f netcoreapp2.1
+    ```
 
 * From another terminal, run the client:
 
-```sh
-> cd GreeterClient
-> dotnet run -f netcoreapp2.1
-```
+    ```sh
+    > cd GreeterClient
+    > dotnet run -f netcoreapp2.1
+    ```
```

Comment:
Nit: I'd indent only 2 spaces here so that the bullet text and code are aligned.

Reply:
Without indenting four spaces, this actually will *not* be aligned, as you can see here: https://grpc.io/docs/quickstart/csharp/

Reply:
Two spaces are enough. E.g., the last bullet of [Objective-C Quick Start > Prerequisites](https://grpc.io/docs/quickstart/objective-c/#prerequisites) is [indented only two spaces](https://github.com/grpc/grpc.io/pull/112/files#r390515484):

> ![image](https://user-images.githubusercontent.com/4140793/76345253-5d3e2300-62d9-11ea-922f-014570a9e637.png)

It renders just fine:
> ![image](https://user-images.githubusercontent.com/4140793/76345186-439cdb80-62d9-11ea-8998-52799b5ce80f.png)

Reply:
In the section that you pointed to in the doc, the indentation is broken: https://grpc.io/docs/quickstart/csharp/#run-a-grpc-application

Reply:
Fixed